fdbserver/workloads/TargetedKill.actor.cpp

```diff
@@ -99,6 +99,17 @@ struct TargetedKillWorkload : TestWorkload {
 				o = ++o%proxies->size();
 			}
 		}
+		else if( self->machineToKill == "grvproxy" ) {
```

Comment:
Interesting. Where do we set different types of `machineToKill`?

Reply:
I think it is supposed to be set in some test configuration file, but I couldn't find any current test that triggers this.